assets/global.css.liquid

```diff
@@ -23,6 +23,95 @@ html {
   --brand-cta-green: #47F5A3;
   --brand-cream: #F0EFEF;
 }
+
+
+/* Font Declarations */
+@font-face {
+  font-family: 'Basic Sans';
+  src: url({{ 'BasicSans-Regular.woff2' | asset_url }}) format('woff2'),
+      url({{ 'BasicSans-Regular.woff' | asset_url }}) format('woff');
+  font-weight: normal;
+  font-style: normal;
+  font-display: swap;
+}
+
+@font-face {
+  font-family: 'BasicSans';
+  src: url({{ 'BasicSans-Bold.woff2' | asset_url }}) format('woff2'),
+      url({{ 'BasicSans-Bold.woff' | asset_url }}) format('woff');
+  font-weight: bold;
+  font-style: normal;
+  font-display: swap;
+}
+
+
+
+@font-face {
+  font-family: 'BasicSans';
+  src: url({{ 'BasicSans-Light.woff2' | asset_url }}) format('woff2'),
+      url({{ 'BasicSans-Light.woff' | asset_url }}) format('woff');
+  font-weight: 300;
+  font-style: normal;
+  font-display: swap;
+}
+
```

Comment:
why is it needed to add the typeface files. I believe these are externally included via Adobe Fonts / Typekit in theme.liquid:
```
<link rel="stylesheet" href="https://use.typekit.net/zoq0xkd.css">
```